pkg/extension/opampextension/config.go

```diff
@@ -34,6 +36,9 @@ type Config struct {
 	// RemoteConfigurationDirectory is where received OpAMP remote configuration
 	// is stored.
 	RemoteConfigurationDirectory string `mapstructure:"remote_configuration_directory"`
+
+	// AcceptsRemoteConfiguration indicates if the OpAMP agent will accept remote configuration.
+	AcceptsRemoteConfiguration bool `mapstructure:"accepts_remote_configuration"`
```

Comment:
Is there a scenario when someone does want to use `opamextension` but dos not want to accept remote configuration?

Reply:
Yes. OpAMP can be used for other capabilities such as heartbeats, health, and effective configuration reporting. Core collection wanted this capability to be configurable at the start, before we build out the other capabilities.